src/main/java/seedu/address/logic/commands/deliverable/ViewCommand.java

```diff
@@ -18,7 +18,8 @@
     public static final String COMMAND_WORD = "view";
     public static final String MESSAGE_VIEW_DELIVERABLE_SUCCESS = "Viewing deliverable: %1$s";
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Views the details of the deliverable identified by the index number used in the deliverable list.\n"
+            + ": Views the details of the deliverable identified by the index number used in "
```

Comment:
May want to consider replacing "used" with "as shown"/"as seen"?

Reply:
Original phrasing of AB3, thinking it would better to just stick to that